src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ArrayConverter.cs

```diff
@@ -33,7 +32,81 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
 
             return base.ConvertTo(context, culture, value, destinationType);
         }
-    }
-}
 
+#if !NETSTANDARD10
+        /// <devdoc>
+        ///    <para>Gets a collection of properties for the type of array specified by the value parameter.</para>
+        /// </devdoc>
+        public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContext context, object value, Attribute[] attributes)
+        {
+            PropertyDescriptor[] props = null;
+
+            if (value.GetType().IsArray)
+            {
+                Array valueArray = (Array)value;
+                int length = valueArray.GetLength(0);
```

Comment:
I think we should leave this as is because our goal is compatibility. Existing code won't be broken by this method's continuing undefined behavior for such cases.